src/main/java/com/google/sps/servlets/ListDrugInfoServlet.java

```diff
@@ -35,38 +32,36 @@
 
 /** Servlet responsible for getting drug info. */
 @WebServlet("/list-drug-info")
-public class DrugServlet extends HttpServlet {
+public class ListDrugInfoServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
-    String drugname = request.getParameter()
-    Query<Entity> query =
-        Query.newEntityQueryBuilder().setKind("Drug").setOrderBy(OrderBy.asc("Name")).build();
+    String gqlQuery = "SELECT * FROM Drug WHERE Name = " + request.getParameter("Name");
```

Comment:
I think the parameter is sending is the id

Reply:
oh okay, fixing it now

Reply:
Sorry, thats outdated check Files Changed for the current change.

Reply:
okay